src/session_state.c

```diff
@@ -705,6 +747,10 @@ void session_state_serialize_prepare_free(Textsecure__SessionStructure *session_
         free(session_structure->alicebasekey.data);
     }
 
+    if(session_structure->has_alicesbuf) {
```

Comment:
Is there a reason why the other buffers are not `free`d as well?

Reply:
freed up c, Xfull, and Rfull buffers